src/internal/tab.ts

```diff
@@ -64,6 +64,19 @@ export class PlaygroundInternalTab extends LitElement {
     button:active::before {
       opacity: 20%;
     }
+
+    :host([active]) > button::after {
```

Comment:
I didn't check this manually but it looks good. A screenshot may help me better understand this. Is `:host` implicitly create a new stacking context for the `position: absolute;`?
Thank you!

Reply:
Before:

![image](https://user-images.githubusercontent.com/48894/127386447-d0e46d81-cc3b-4aa7-bd6a-58bca3942731.png)

After:

![image](https://user-images.githubusercontent.com/48894/127386588-a6a33701-3296-4ef0-88d9-0d55a4865f0e.png)

This is the state when the index.html tab is hovered. Note the white gap under it. (font-family also changed because of the other fix).

The `::after` pseudo-element does get a new stacking context since it's `absolute`, yeah. `<mwc-tab>` (which we used before https://github.com/PolymerLabs/playground-elements/pull/159) actually uses an absolutely positioned element for the indicator.